src/sortStudents.ts

```diff
@@ -1,16 +1,62 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function averageGrade(arrayOfGrades : number[]) : number {
+  const sum = arrayOfGrades.reduce((a : number, b : number) => a + b, 0);
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  return sum / arrayOfGrades.length;
+}
+
+// eslint-disable-next-line max-len
+export function sortStudents(students : Student[], sortBy : SortType, orderBy : SortOrder) : Student[] {
```

Comment:
If characters length in line greater than 80, or number of parameters grater than 3, split it into rows
```suggestion
export function sortStudents(
  students : Student[], 
  sortBy : SortType, 
  orderBy : SortOrder
) : Student[] {
```